crayfish/rootfs/etc/nginx/http.d/default.conf

```diff
@@ -3,6 +3,10 @@ server {
     listen 8000;
     root /var/www/html;
 
+    # have docker logs show the client IP
+    real_ip_header X-Forwarded-For;
+    set_real_ip_from 172.0.0.0/8;
```

Comment:
So this can be other values, so we should ideally make this configurable via an environment variable. I believe both `real_ip_header` and `set_real_ip_from` can be used in the `http` block. So we can apply this globally by changing the nginx image.

So in this [block](https://github.com/Islandora-Devops/isle-buildkit/blob/95ecba31b77c011e36afe8358b64e492332f9d7d/nginx/rootfs/etc/confd/templates/nginx.conf.tmpl#L24). Since this is a template you can then use enviroment variables to populate it so it looks something like.

```nginx
real_ip_header X-Forwarded-For;
set_real_ip_from {{ getenv "NGINX_SET_REAL_IP_FROM" }};
```

And in the [Dockerfile](https://github.com/Islandora-Devops/isle-buildkit/blob/95ecba31b77c011e36afe8358b64e492332f9d7d/nginx/Dockerfile#L61). We add the environment variable with the new default:

```bash
 NGINX_SET_REAL_IP_FROM = 172.0.0.0/8 \
```

Reply:
This would remove the need for changes in `default.conf`

Reply:
> Is Docker always guaranteed to set up the bridge network at 172.0.0.0/8? I'm not sure we'd have the same value for things running in Fargate for example? Or if using host only networking. I'm away for the next few days but I can do some digging next Thursday.

So an example of what DSpace does https://github.com/DSpace/DSpace/blob/main/docker-compose.yml#L2-L8 (to force Docker to have a specific network), so at least that is consistent (this is probably a chance in isle-dc) than here.

Reply:
Good call, @nigelgbanks - I updated the PR accordingly and added a test